MetaMask/metamask-extension
# It should not be possible for user to enable both openSea and blockaid security providers

## **Description**
Make it only possible to have 1 security alert provider enabled

## **Manual testing steps**

_1. Run extension flask build
_2. Go to settings -> Experimental
_3. Toggle blockaid / opensea buttons
_4. Notice that it is not possible to enable both the security providers

## **Screenshots/Recordings**

https://github.com/MetaMask/metamask-extension/assets/2182307/2b154d9e-f7cb-4627-b619-2c16d8fcc27e

## **Related issues**

_Fixes MetaMask/MetaMask-planning#1356

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
